src/pyobo/extract.py

```diff
@@ -183,6 +185,19 @@ def _df_getter() -> pd.DataFrame:
     return _df_getter()
 
 
+def get_id_to_alts(prefix: str, **kwargs) -> Mapping[str, List[str]]:
```

Comment:
This mapping is most often done in the other direction, where, some source provides content that has outdated, withdrawn or secondary IDs (very common with CHEBI, UP and GO) and one needs to map it to the current primary ID. I can add fupport for the reverse mapping if you agree.

Reply:
Addressed in 945873d